src/content/docs/guides/platform-setup/configure-identity-provider.mdx

```diff
@@ -56,25 +71,6 @@ with the OIDC issuer and audiences that the platform should accept.
 TENZIR_PLATFORM_OIDC_TRUSTED_AUDIENCES='{"https://my.idp.example": ["tenzir-cli", "tenzir-app"]}'
 ```
 
-You must configure the set of `admin` users in your platform instance. An admin
-user is a user who is permitted to run the `tenzir-platform admin` CLI command.
-
-The `TENZIR_PLATFORM_OIDC_ADMIN_RULES` setting contains a JSON list of access
-rules that determine who is considered an admin for this platform instance. If
-any of the provided rules match, the platform considers the user an admin. The
-example rule grants admin access to all users with a valid and signed `id_token`
-containing the fields `{"tenzir/org": "TenzirPlatformAdmins"}`.
-
-```sh
-TENZIR_PLATFORM_OIDC_ADMIN_RULES='[{"organization_claim": "tenzir/org", "organization": "TenzirPlatformAdmins", "auth_fn": "auth_organization"}]'
-```
-
-See the documentation on [Access
-Rules](/reference/platform/command-line-interface#configure-access-rules) for
-more information about the possible types of rules and their syntax. Use the
-`tenzir-platform tools` subcommand of the CLI to generate valid JSON objects
-that you can enter here.
-
```

Comment:
Is this information no longer needed or accurate?

Reply:
It's in the "configure internal services" part now, since it's not related to the identity provider.